web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -182,10 +187,10 @@ def post_description(classname):
                      RESOURCE_MUST_EXIST.format(classname=classname)])
 
 
-def object_id_field(klass):
+def object_id_field(klass, type="integer"):
```

Comment:
`type` is a builtin python function, but either way I don't understand why this parameter exist, it is not used at all. If it meed exist for future work, the name of the kwarg should change slightly.

Reply:
It was a left over from an approach I abandoned.  Removed.